meson_options.txt

```diff
@@ -252,5 +252,7 @@ option('zshcompletiondir', type : 'string',
 
 option('tests', type : 'combo', choices : ['true', 'unsafe'],
        description : 'enable extra tests with =unsafe')
+option('slow-tests', type : 'boolean', value : 'false',
+       description : 'run the slow tests by default')
```

Comment:
OK. How about we configure one of the semaphore tests to use slow tests, so that we at least have one code path with them?